# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/PNecko.ipdl
# Commit: 6e1eba5c88ad
# Full Hash: 6e1eba5c88ad47f0a2a3aa37b4c2cae9f31c2ef3
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 1
# Description:
#   Bug 1569183: Stop doing a proxy lookup to determine whether we're configured to use a proxy (for the proxy_only_if_behind_proxy pref), and instead look at whether we loaded the doc using a proxy. r=mjf,mayhemer,jld
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45289
# ==============================================================================

diff -r c43589fd0f90 -r 6e1eba5c88ad netwerk/ipc/PNecko.ipdl
--- a/netwerk/ipc/PNecko.ipdl	Tue Sep 17 18:18:33 2019 +0000
+++ b/netwerk/ipc/PNecko.ipdl	Wed Sep 18 14:27:42 2019 +0000
@@ -28,7 +28,6 @@
 include protocol PClassifierDummyChannel;
 include protocol PWebrtcTCPSocket;
 include protocol PSocketProcessBridge;
-include protocol PProxyConfigLookup;
 include protocol PDocumentChannel;
 
 include IPCStream;
@@ -69,7 +68,6 @@
   manages PStunAddrsRequest;
   manages PClassifierDummyChannel;
   manages PWebrtcTCPSocket;
-  manages PProxyConfigLookup;
   manages PDocumentChannel;
 
 parent:
@@ -146,8 +144,6 @@
   /* tabId is only required for web-proxy support, which isn't always needed */
   async PWebrtcTCPSocket(TabId? tabId);
 
-  async PProxyConfigLookup();
-
   /**
    * WebExtension-specific remote resource loading
    */